src/shared/extensions/yaml.ts

```diff
@@ -49,7 +49,7 @@ export async function activateYamlExtension(): Promise<YamlExtension | undefined
         },
         uri => {
             try {
-                return readFileSync(vscode.Uri.file(uri).fsPath).toString()
+                return readFileSync(vscode.Uri.parse(uri).fsPath).toString()
             } catch (e) {
                 getLogger().error(`YAML Extension: failed to read schema URI "${uri}": ${e}`)
                 throw new Error(`The AWS Toolkit was unable to parse the JSON schema URI: ${uri}`)
```

Comment:
if this is potentially user-facing we need to replace "AWS" with China-region name, e.g.: 

https://github.com/aws/aws-toolkit-vscode/blob/7fbc3c3d85ef346e5c7203d9de5d54f8fb23b8a0/src/awsexplorer/awsExplorer.ts#L65-L66

Reply:
It could potentially be user-facing through the YAML extension if they decide to show the error message.